Doofinder/Feed/Model/Data/InstallationStruct.php

```diff
@@ -80,51 +94,101 @@ public function __construct(
         $this->pluginVersion = $pluginVersion;
     }
 
+    /**
+     * Get the name of the installation.
+     *
+     * @return string Name of the installation.
+     */
     public function getName(): string
     {
         return $this->name;
     }
 
+    /**
+     * Get the platform of the installation.
+     *
+     * @return string Platform of the installation.
+     */
     public function getPlatform(): string
     {
         return $this->platform;
     }
 
+    /**
+     * Get the primary language of the installation.
+     *
+     * @return string Primary language of the installation.
+     */
     public function getPrimaryLanguage(): string
     {
         return $this->primaryLanguage;
     }
 
+    /**
+     * Get whether to skip indexation.
+     *
+     * @return bool Whether to skip indexation.
+     */
     public function getSkipIndexation(): bool
     {
         return $this->skipIndexation;
     }
 
+    /**
+     * Get the sector of the installation.
+     *
+     * @return string Sector of the installation.
+     */
     public function getSector(): string
     {
         return $this->sector;
     }
 
+    /**
+     * Get the site URL of the installation.
+     *
+     * @return string Site URL of the installation.
+     */
     public function getSiteUrl(): string
     {
         return $this->siteUrl;
     }
 
+    /**
+     * Get the search engines associated with the installation.
+     *
+     * @return array Search engines associated with the installation.
```

Comment:
We must avoid using array as return type, otherwise the internal swagger will fail.

```suggestion
     * @return mixed[] Search engines associated with the installation.
```